.github/workflows/sync-teams.yml

```
@@ -0,0 +1,41 @@
+name: '[Support] Synchronize team members in the .env file'
+on:
+  schedule:
+    # Daily
+    - cron: '0 1 * * *'
+permissions:
+  repository-projects: write
+
+jobs:
+  sync-support-teams:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Repo checkout
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.BITNAMI_BOT_TOKEN }}
+          fetch-depth: 1
+      - name: Load .env file
+        uses: xom9ikk/dotenv@v1.0.2
+        with:
+          path: .github/workflows/
+      - name: Updating members of the Bitnami team
+        run: |
+          TEAM_MEMBERS=$(curl --request GET \
+          --url https://api.github.com/repos/orgs/bitnami/teams/developers/members \
+          --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
+          --header 'content-type: application/json' \
+          | jq 'sort_by(.login)|map(.login)|join(",")') \
+          TEAM_MEMBERS='['${TEAM_MEMBERS//','/'","'}']'
+          if [ $TEAM_MEMBERS != $BITNAMI_TEAM ]; then
+            echo "Replacing $BITNAMI_TEAM for $TEAM_MEMBERS"
+            cp .github/workflows/.env .github/workflows/.env-copy
+            sed '1d' .github/workflows/.env-copy > .github/workflows/.env
+            echo "BITNAMI_TEAM='$TEAM_MEMBERS'" | cat - .github/workflows/.env > .github/workflows/.env-copy && mv .github/workflows/.env-copy .github/workflows/.env
```

Not fully tested but I think you can use this:

```suggestion
            sed -i "s|BITNAMI_TEAM=.*$|BITNAMI_TEAM='${TEAM_MEMBERS}'|g" .github/workflows/.env
```

---

I think I tested it and failed. Let me try again

---

You were fine, I'll update the workflow with that suggestion. Thanks!